core/src/main/java/de/unistuttgart/iaas/amyassist/amy/core/natlang/nl/NLParser.java

```diff
@@ -49,6 +49,8 @@
 
 	private int currentIndex;
 
+	private Stemming stemmer = new Stemming();
```

Comment:
the stemmer should be received from LanguageSpecifics, because the Stemming class only works for the english language